src/GitTfs.VsCommon/TfsHelper.Common.cs

```diff
@@ -219,7 +219,7 @@ public virtual int FindMergeChangesetParent(string path, int targetChangeset, Gi
 
             var changes = VersionControl.GetChangesForChangeset(targetChangeset, false, Int32.MaxValue, null, null, true).Where(change =>
             {
-                return change.ChangeType.HasFlag(ChangeType.Merge) && change.Item.ServerItem.Contains(path);
+                return change.ChangeType.HasFlag(ChangeType.Merge) && change.Item.ServerItem.StartsWith(path, StringComparison.OrdinalIgnoreCase);
```

Comment:
It fixes a problem with a specific TFS project in my company that could not be `git-tfs-clone`’d; after I put this change in, I was able to clone.  Also, I believe that before the commit https://github.com/git-tfs/git-tfs/commit/742a29f5eb9e4a6117988147f96d58f3337f2575 was put in, my project would clone successfully, so it may be considered a regression (I have not tested it, though.)

Let me put it in a bit of context: we have an in-house version-control tool that talks to TFS via TeamFoundation API like git-tfs does, and as such, it is a bit rough around the edges, so maybe this problem would not be reproducible in projects created via "official" Visual Studio tooling...   One issue that we see with it is that in some cases, when you retrieve files they have ALL-CAPS names even if the developer typed the name in CamelCase originally (`"DIRBLAH\FILEBLAH.CPP"` instead of `"DirBlah\FileBlah.cpp"`.)  It worked like that for a long time, and we have never had issues with that, so I think TFS follows Microsoft Windows conventions here and ignores case differences in file paths.

When I put a breakpoint at this code, what I saw was:
```
path = "$/Company Project"
change.Item.ServerItem = "$/COMPANY Project/DirBlah/FileBlah.cpp"
```
After this fix, I was able to do `git-tfs clone` successfully.  Then, when I run `git-tfs verify` I get a bunch of diagnostics like this, which seem acceptable:
```
Comparing TFS changeset 1331335 to git commit 8c4e0aac103cd664190dac4b30498192a529ac70
...
Name case mismatch:
  TFS: BlahLib/ControllerConnection/BlahFile.cs
  git: BlahLib/CONTROLLERCONNECTION/BLAHFILE.CS
...
```
and the resulting git repository seem to be in a good shape as far as I can tell.

Reply:
Would [`IndexOf`](https://docs.microsoft.com/en-us/dotnet/api/system.string.indexof) work instead?

```suggestion
                return change.ChangeType.HasFlag(ChangeType.Merge) && (change.Item.ServerItem.IndexOf(path, StringComparison.OrdinalIgnoreCase) >= 0);
```

Reply:
> Would [`IndexOf`](https://docs.microsoft.com/en-us/dotnet/api/system.string.indexof) work instead?

Based on what I observed—granted it is only a single data point—it logically must be `StartsWith`.  `IndexOf` would happen to work too, because the symbol `$` in TFS path occurs only at the beginning, but that would be a waste of cycles if my understanding that `path` is always an absolute path is correct.

What I was less sure about is which `StringComparison.*IgnoreCase` option exactly would be correct—they are all over the place in the existing code.